languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -8788,6 +8788,22 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rule id="LA_OU_LES_PLUS" name="la ou les plus" default="temp_off">
+          <pattern>
+              <token postag="V.*" postag_regexp="yes"></token>
+              <marker>
+                  <token regexp="yes">la|les</token>
+              </marker>
+              <token regexp="yes">plus|moins</token>
+              <token min="0" max="2" postag="A.*" postag_regexp="yes"></token>
+              <token>.</token>
+          </pattern>
+          <message>« la|les» est une expression fautive.</message>
```

Comment:
`<message>Il semble qu'il ait une faute dans l'article '\2'... </message>`

Reply:
Please do not put quotation marks around the `<suggestion>` — quotation marks are added automatically to that element.